valhalla/filesystem.h

```diff
@@ -359,4 +422,7 @@ inline bool remove_all(const path& p) {
   return remove(p);
 }
 
+inline path unique_path() {
+  return path("TODO:");
```

Comment:
this should be the only thing left to implement other than the unit tests for it